src/allotropy/allotrope/models/shared/definitions/units.py

```diff
@@ -1,188 +1,239 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
NOTE: This file is now auto-generated and alphabetized, hence the large amount of diffs